### PR TITLE
Initial draft of NVM installation and usage guide

### PR DESCRIPTION
**Pull request by JeffreyNovotny**
Guide for NVM installation and usage guide.

### CODE REVIEW

**Comment by nmelehan** [comment]
:heavy_check_mark: Deploy preview for *nostalgic-ptolemy-b01ab8* ready!


:hammer: Explore the source changes: efb0ef73d0e0c9268485a8a525ddc077ca47220e

:mag: Inspect the deploy logs: [https://app.netlify.com/sites/nostalgic-ptolemy-b01ab8/deploys/6000422d907c650008c4fc3d](https://app.netlify.com/sites/nostalgic-ptolemy-b01ab8/deploys/6000422d907c650008c4fc3d?utm_source=github&utm_campaign=bot_dl)

:sunglasses: Browse the preview: [https://deploy-preview-3903--nostalgic-ptolemy-b01ab8.netlify.app](https://deploy-preview-3903--nostalgic-ptolemy-b01ab8.netlify.app?utm_source=github&utm_campaign=bot_dp)

File: docs/guides/development/nodejs/how-to-install-use-node-version-manager-nvm/index.md [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+*   NVM_INC: The directory for Node's include files (for building C/C++ add-ons).
+*   NVM_CD_FLAGS: The flags used to maintain compatibility with `zsh`.
+*   NVM_RC_VERSION: The version from the `.nvmrc` file (if used).
+    {{< /note >}}
+3.  Confirm you have successfully installed NVM.
+
+        command -v nvm
+    If the installation was successful, NVM returns `nvm`. If you see the message `nvm: command not found`, confirm the original installation completed successfully and repeat step #2 of this section to source the `.bashrc` file.
+
+        nvm
```
Comment (andystevensname) [inline docs/guides/development/nodejs/how-to-install-use-node-version-manager-nvm/index.md:89]: Use  the {{< output >}} tag to denote output

File: docs/guides/development/nodejs/how-to-install-use-node-version-manager-nvm/index.md [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+## Using NVM to Install Node
+
+The `install` command downloads, compiles, and installs the specified version of Node. You can install as many versions of Node as you want.
+
+1.  To install the current version of Node, run the following:
+
+        nvm install node
+    After a successful install, NVM displays information about the installation.
+
+        Now using node v15.5.1 (npm v7.3.0)
```
Comment (andystevensname) [inline docs/guides/development/nodejs/how-to-install-use-node-version-manager-nvm/index.md:103]: Use the output tag

File: docs/guides/development/nodejs/how-to-install-use-node-version-manager-nvm/index.md [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+        Creating default alias: default -> node (-> v15.5.1)
+    {{< note >}}
+When used in a NVM command, `node` is an alias for the latest version of Node. The first version of Node you installed automatically becomes the default version. A new shell instantiates with the current version of Node set to `default`. The alias `system` refers to the system-installed version of Node (if any).
+    {{< /note >}}
+2.  To install a specific version of Node, specify the major or minor release number. You can preview a list of all available Node versions with the `ls-remote` command.
+
+        nvm ls-remote
+    NVM displays a long list of available versions in the following format:
+
+        v5.12.0
```
Comment (andystevensname) [inline docs/guides/development/nodejs/how-to-install-use-node-version-manager-nvm/index.md:113]: Use  the output tag

File: docs/guides/development/nodejs/how-to-install-use-node-version-manager-nvm/index.md [hunk 4]
```diff
@@ -0,0 +1,227 @@
...
+   {{< /note >}}
+
+## Using NVM to Run Node
+
+1.  Review all installed versions of Node with the `ls` command.
+
+        nvm ls
+    NVM returns a list of all Node versions and aliases, along with an arrow indicating the current version. Here is a sample output:
+
+        v13.10.1
```
Comment (andystevensname) [inline docs/guides/development/nodejs/how-to-install-use-node-version-manager-nvm/index.md:130]: Use the  output   tag

File: docs/guides/development/nodejs/how-to-install-use-node-version-manager-nvm/index.md [hunk 5]
```diff
@@ -0,0 +1,227 @@
...
+        stable -> 15.5 (-> v15.5.1) (default)
+2.  To select a different version of Node, use the `nvm use` command. Specify either the version number of Node (major or minor release) or an alias such as `node`.
+
+        nvm use node
+    or
+
+        nvm use 14
+    NVM confirms it is now using the new version.
+
+        Now using node v14.15.4
```
Comment (andystevensname) [inline docs/guides/development/nodejs/how-to-install-use-node-version-manager-nvm/index.md:145]: Use  the output tag

File: docs/guides/development/nodejs/how-to-install-use-node-version-manager-nvm/index.md [hunk 6]
```diff
@@ -0,0 +1,227 @@
...
+    You can also confirm the current version of Node with `nvm current`.
+
+        nvm current
+    NVM again returns the current version number.
+
+        v14.15.4
```
Comment (andystevensname) [inline docs/guides/development/nodejs/how-to-install-use-node-version-manager-nvm/index.md:151]: Output tag

File: docs/guides/development/nodejs/how-to-install-use-node-version-manager-nvm/index.md [hunk 7]
```diff
@@ -0,0 +1,227 @@
...
+
+    {{< note >}}
+You can also confirm the version of Node currently in use with `node -v`. To go back to using the system's version of Node, run the command `nvm use system`.
+    {{< /note >}}
+3.  To switch to a new version of Node and immediately open a Node console, use `nvm run`. The `run` command is very similar to `nvm use` in all other respects.
+
+        nvm run node
+    NVM confirms it is now running the selected version of Node and returns a Node prompt.
+
+        Running node v15.5.1 (npm v7.3.0)
```
Comment (andystevensname) [inline docs/guides/development/nodejs/how-to-install-use-node-version-manager-nvm/index.md:161]: Output tag

File: docs/guides/development/nodejs/how-to-install-use-node-version-manager-nvm/index.md [hunk 8]
```diff
@@ -0,0 +1,227 @@
...
+        nvm alias default 14
+    NVM confirms the new value for the alias.
+
+        default -> 14 (-> v14.15.4)
+2.  Use the `nvm alias` command to create a new alias. The following example defines a new `maintenance` alias as Node version 13.10.1.
+
+        nvm alias maintenance 13.10.1
+    NVM confirms the new alias.
+
+        maintenance -> 13.10.1 (-> v13.10.1)
```
Comment (andystevensname) [inline docs/guides/development/nodejs/how-to-install-use-node-version-manager-nvm/index.md:180]: Use output  tag

File: docs/guides/development/nodejs/how-to-install-use-node-version-manager-nvm/index.md [hunk 9]
```diff
@@ -0,0 +1,227 @@
...
+## Creating NVM Aliases
+
+You might find it convenient to refer to a Node version by a different or easier to remember name. NVM already provides some pre-made defaults such as `default` and `node`, which refers to the latest version. But you can use the `alias` command to change the value of an existing alias or create a brand-new alias.
+
+1.  Use the `nvm alias` command to change the default Node version. Follow the `alias` keyword with the `default` alias and the new version of Node it should reference.
+
+        nvm alias default 14
+    NVM confirms the new value for the alias.
+
+        default -> 14 (-> v14.15.4)
```
Comment (andystevensname) [inline docs/guides/development/nodejs/how-to-install-use-node-version-manager-nvm/index.md:174]: Use output tag

**Review by andystevensname — CHANGES_REQUESTED** [review]

**Review by rsyracuse — APPROVED** [review]

**Review by andystevensname — APPROVED** [review]

**Pull request merged**